zask/__init__.py

```diff
@@ -40,10 +40,14 @@ class LocalContext(object):
     """
     Store data in local greenlet context.
 
-    > Gevent also allows you to specify data which is local to the greenlet context.
-    > Internally, this is implemented as a global lookup which addresses a private namespace keyed by the greenlet's getcurrent() value.
+    > Gevent also allows you to specify data which is local to
+      the greenlet context.
+    > Internally, this is implemented as a global lookup
+      which addresses a private namespace keyed by the
+      greenlet's getcurrent() value.
```

Comment:
This doesn't render correctly when building sphinx documentation:

![screen shot 2016-08-02 at 6 25 01 pm](https://cloud.githubusercontent.com/assets/2793514/17325507/8929aa88-58de-11e6-9f36-dbd3f7b2b439.png)